libs/filabridge/include/private/filament/UibStructs.h

```diff
@@ -161,8 +161,19 @@ struct PerViewUib { // NOLINT(cppcoreguidelines-pro-type-member-init)
     float ssrDistance;                  // ssr world raycast distance, 0 when ssr is off
     float ssrStride;                    // ssr texel stride, >= 1.0
 
+    // --------------------------------------------------------------------------------------------
+    // Localized, tinted IBL parameters
+    // --------------------------------------------------------------------------------------------
+    math::float3 iblCenter;           // center of the sphere or IBL AABB
```

Comment:
This change will require to bump the materials version number as it creates a binary incompatibility.

@bejado